functions/ts/package.json

```diff
@@ -30,12 +30,15 @@
   "devDependencies": {
     "@types/jasmine": "^3.5.14",
     "@types/js-yaml": "^3.12.5",
-    "@types/node": "^14.11.2",
+    "@types/node": "^14.11.10",
```

Comment:
If you haven't already take a look at the guideline for adding dependencies (even if they're devDependencies):

https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/CONTRIBUTING.md#adding-an-npm-dependency

Even though it's very easy to add dependencies with tools like NPM, each new dependency needs to be scrutinized.

Reply:
[`nyc`](https://www.npmjs.com/package/nyc) is the CLI interface for unit test coverage tool [`istanbuljs`](https://istanbul.js.org/). `nyc` is a big project and I cannot tell that all dependencies are high quality but it is one of the best tool that I can found for showing coverage report. It's a builtin component in [`jest`](https://github.com/facebook/jest) as well.
[`proxyquire`](https://www.npmjs.com/package/proxyquire) is used to do mock in the unit test. It's very simple and has only seldom dependencies.
Their licenses are OK.

I need to add these dependencies since `jasmine` doesn't have enough capabilities (coverage report and function/class mock).

Reply:
proxyrequire has fewer deps:

http://npm.broofa.com/?q=proxyquire

It should be fine.